sdk/netapp/azure-mgmt-netapp/dev_requirements.txt

```diff
@@ -1,4 +1,5 @@
 -e ../../../tools/azure-sdk-tools
 -e ../../../tools/azure-devtools
 -e ../../../sdk/identity/azure-identity
--e ../../network/azure-mgmt-network
\ No newline at end of file
+-e ../../network/azure-mgmt-network
+-e ../../resources/azure-mgmt-resource
```

Comment:
@audunn Why do you remove dependency for this package?
![image](https://user-images.githubusercontent.com/70930885/197663745-ede5664c-a174-419c-90d8-fbfb843941e0.png)

Reply:
I did not remove, I am adding this to dev_requrements. 

https://teams.microsoft.com/l/message/19:59dbfadafb5e41c4890e2cd3d74cc7ba@thread.skype/1666633095067?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&parentMessageId=1666386449379&teamName=Azure%20SDK&channelName=Engineering%20System%20%F0%9F%9B%A0%EF%B8%8F&createdTime=1666633095067&allowXTenantAccess=false